src/components/Disputes/Import.js

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable  */
+import React, { useState, useEffect } from 'react'
+import { SidePanel, Button, DropDown, TextInput, textStyle } from '@aragon/ui'
+import { Octokit } from '@octokit/rest'
+import { useWallet } from 'use-wallet'
+
+function Import() {
+  const wallet = useWallet()
+
+  const [USERNAME, SETUSERNAME] = useState(null)
+
+  const [opened, setOpened] = useState(false)
+
+  const [list, setList] = useState([])
+
+  const TOKEN = localStorage.getItem('ACCESS TOKEN')
+
+  let USERNAMETEST = localStorage.getItem('USERNAME')
+
+  const octokit = new Octokit({
+    auth: TOKEN,
+  })
+
+  // async function userInfo() {
+  //   const { data } = await octokit.request('/user')
+  //   console.log(data.login)
+  //   SETUSERNAME(data.login)
+  //   return data.login
+  // }
+  // userInfo()
+
+  async function handleButtonClick() {
+    console.log(localStorage.getItem('ACCESS TOKEN'))
+
+    setOpened(true)
+    await octokit.repos
+      .listForUser({
+        // username: 'abhinav-anshul',
+        username: USERNAMETEST,
+      })
+
+      .then((details) => {
+        let x = 0
```

Comment:
Name the variables relevant to the functionality:
Ex: `repoIndex` instead of `x`, `repoList` instead of `arr`

Reply:
OUTDATED

Reply:
This is not outdated. Needs to be fixed.

Reply:
Fixed